src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaTestHelpers.cs

```diff
@@ -388,7 +397,7 @@ internal static byte[] DoTryUntilDone(TryExportFunc func)
                 "SLH-DSA-SHAKE-192f" => "2.16.840.1.101.3.4.3.29",
                 "SLH-DSA-SHAKE-256s" => "2.16.840.1.101.3.4.3.30",
                 "SLH-DSA-SHAKE-256f" => "2.16.840.1.101.3.4.3.31",
-                _ => null,
+                _ => throw new XunitException("Unknown algorithm."),
```

Comment:
```suggestion
                _ => throw new XunitException($"Unknown algorithm '{algorithm?.Name}'"),
```

Seems better to me...